src/governor.cairo

```diff
@@ -130,6 +139,7 @@ mod Governor {
                 proposal.no = proposal.no + weight;
             }
             self.proposals.write(id, proposal);
+            self.voted.write((voter, id), true);
```

Comment:
not using the mut pattern here. i'd say just load it immutable or dont assign it

Reply:
yep right. loaded it immutable